reggiedata/patches/NewerSMBW/spritedata.xml

```diff
@@ -1863,774 +862,49 @@
-      <entry value="0">Reset Sprite Settings!</entry>
-      <entry value="5">Really Not Recommended</entry>
-      <entry value="6">Slow (Standard) (Not Recommended)</entry>
-      <entry value="7">Medium (Not Recommended)</entry>
-      <entry value="4">Really Not Recommended</entry>
-      <entry value="1">Really Not Recommended</entry>
-      <entry value="2">Really Not Recommended</entry>
-      <entry value="3">Really Not Recommended</entry>
-    </list> <!-- This version of the list shows when Falling Lava is set, and Distance is also is set. You bad bad boy... -->
-   <!-- Rising vs Falling Options -->
-    <dualbox bit="17" title1="[???] Rising Forever" title2="[???] Falling Forever" comment="This determines which direction the Lava will move." requiredbit="18-20, 29-36, 21-28, 1-8" requiredval="1-7, 0, 0, 0"/> <!-- Appears when Speed is set and Distance, Delays, and Triggering Event ID are NOT set -->
-    <dualbox bit="17" title1="[???] Rising Forever while On" title2="[???] Falling Forever while On" comment="This determines which direction the Lava will move while the Triggering Event is activated." requiredbit="18-20, 29-36, 21-28, 1-8" requiredval="1-7, 0, 0, 1-255"/> <!-- Appears when Speed and Triggering Event ID are set and Distance is NOT set -->
-    <dualbox bit="17" title1="[???] Automatically Rising and Falling" title2="[Extremely Buggy]" comment="The Lava will automatically rise and fall in accordance with the set delays." requiredbit="18-20, 29-36, 1-8" requiredval="1-7, 1-255, 0"/> <!-- Appears when Speed and Distance are set, and Triggering Event ID is not set. -->
-    <dualbox bit="17" title1="[???] Rises Distance while On, Falls when Off" title2="[Extremely Buggy]" comment="The Lava will rise the Distance value once activated, and stay at the top until the Triggering Event is deactivated. If the Triggering Event is deactivated before it rises the full distance, it will finish rising, then immediately start to fall." comment2="This option only applies if Delay values are both set to 0, Distance is set, and Triggering ID is set. Do not use [Extremely Buggy]. No really, don't. It won't help you any." requiredbit="
```

Comment:
This answers the comment on the line above, so you should probably remove that comment.